pandas/tests/groupby/conftest.py

```diff
@@ -76,3 +76,13 @@ def three_group():
                       'D': np.random.randn(11),
                       'E': np.random.randn(11),
                       'F': np.random.randn(11)})
+
+
+@pytest.fixture
```

Comment:
can this be more generally used in groupby/test_categorical.py?

Reply:
I would have to keep doing `.astype('category')` in each of my tests for this. Either that, or derive another fixture from `three_group`, so it would not decrease the number of fixtures.

I also preferred literal values to random ones for easier equality checks.

Reply:
it’s not about decreasing the number of fixtures
but rather reusing them across tests as much as possible

Reply:
literal values are fine (u could just replace the random with fixed values)
changing existing to accommodate new tests is better than rolling new